apollo-backend/src/main/java/io/logz/apollo/controllers/BlockerDefinitionController.java

```diff
@@ -168,9 +168,13 @@ public void updateBlockerDefinition(int id, String name, Integer environmentId,
                                         Boolean isActive, String blockerTypeName, String blockerJsonConfiguration, Req req) {
         String userEmail = req.token().get("_user").toString();
         logger.info("User: {} has just updated blocker, id:{}, name:{}", userEmail, id, name);
-
         BlockerDefinition blockerDefinition = blockerDefinitionDao.getBlockerDefinition(id);
 
+        if (!blockerDefinition.getBlockerTypeName().equals(blockerTypeName)) {
```

Comment:
potential NPE
this check should be after line 182

Reply:
@matvey-mtn You can not create a blocker without a type. I can change it to after line 182 if you want

Reply:
@maorshap it's not a type
blockerDefinition could be null

Reply:
@matvey-mtn Oh you are talking about blocker Definition.. 
I will add NPE check then :)